lms/djangoapps/onboarding/models.py

```diff
@@ -901,3 +901,42 @@ class RegistrationType(models.Model):
         related_name='registration_type'
     )
     choice = models.SmallIntegerField(default=1, null=False)
+
+
+class Education(models.Model):
+    """
+    Model to store user education information from linkedin
+    """
+    linkedin_id = models.IntegerField(null=True, blank=True)
```

Comment:
Please make this field unique in all 3 models

Reply:
done